kpet/utils.py

```diff
@@ -33,3 +37,106 @@ def get_template_content(filename, dbdir=None):
     template_path = get_template_path(filename, dbdir)
     with open(template_path) as file_handler:
         return file_handler.read()
+
+
+def print2(s):
+    sys.stderr.write(s + '\n')
+
+
+def smart_layout_insert_prefix(f_layout, obj):
+    if obj is None:
+        return None
```

Comment:
Quoting the zen of Python `Errors should never pass silently. Unless explicitly silenced.` Please remove this and use a proper exception handling if it is necessary.

Reply:
I'll like to remove this function to make things simple.